view/statistics-flow-certificates.js

```diff
@@ -140,10 +140,10 @@ exports['statistics-main'] = function () {
 			p({ class: 'submit' }, input({ type: 'submit' }))),
 		section({ class: 'pad-if-pagination' }, pagination),
 		br(),
-		section({ class: "section-primary" },
 			data.map(function (result) {
-				var mode = modes.get(location.query.mode || 'daily');
-				return table({ class: 'statistics-table' },
+			var mode = modes.get(location.query.mode || 'daily');
+			return div({ class: 'overflow-x width-1050px' },
```

Comment:
I'm not sure that those classes are needed. However, what is needed is `responsive-container` class (See rejection reasons table for reference).